openmdao/surrogate_models/kriging.py

```diff
@@ -187,6 +207,11 @@ def predict(self, x, eval_rmse=True):
         # Predictor
         y = self.Y_mean + self.Y_std * y_t
 
+        #Test for EGOLF
```

Comment:
I'd remove these commented-out lines, but if you wanted to, you could add a test to test_kriging.py that verifies this calculation.